### PR TITLE
PE-3217: details panel + share page

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/6ne56v3mked10
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/15vo5do055jr8

### CODE REVIEW

File: lib/components/details_panel.dart [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+            folderCount: folder.entry.getRecursiveSubFolderCount(),
+            fileCount: folder.entry.getRecursiveFileCount(),
+            localizations: appLocalizationsOf(context),
+          ),
+          style: ArDriveTypography.body.buttonNormalRegular(),
+        ),
+        itemTitle: appLocalizationsOf(context).itemContains,
+      ),
+      const SizedBox(
+        height: 16,
```
Comment (karlprieb) [inline lib/components/details_panel.dart:218]: a lot of repeated `SizedBox` height 16. Create a const for this number.

File: lib/pages/drive_detail/components/drive_detail_data_list.dart [hunk 2]
```diff
@@ -3,6 +3,7 @@ part of '../drive_detail_page.dart';
 Widget _buildDataList(BuildContext context, DriveDetailLoadSuccess state) {
   int index = 0;
 
+  // TODO(@thiagocarvalhodev): Refactor using the new method for select items
```
Comment (karlprieb) [inline lib/pages/drive_detail/components/drive_detail_data_list.dart:6]: which method?

File: lib/pages/drive_detail/components/drive_explorer_item_tile.dart [hunk 3]
```diff
@@ -346,3 +354,11 @@ class _DriveExplorerItemTileTrailingState
     );
   }
 }
+
+bool isMobile(BuildContext context) {
+  final screenHeight = MediaQuery.of(context).size.height;
+  final isPortrait = MediaQuery.of(context).orientation == Orientation.portrait;
+  return isPortrait &&
+      screenHeight <
```
Comment (karlprieb) [inline lib/pages/drive_detail/components/drive_explorer_item_tile.dart:362]: we don't need `screenHeight`, `isPortrait` is enough

File: lib/pages/drive_detail/drive_detail_page.dart [hunk 4]
```diff
@@ -80,271 +75,626 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
...
+              padding: const EdgeInsets.symmetric(
+                vertical: 8,
+                horizontal: 16,
+              ),
+              child: Stack(
+                children: [
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      // Column(
```
Comment (karlprieb) [inline lib/pages/drive_detail/drive_detail_page.dart:423]: remove unused comments

File: lib/services/config/app_config.dart [hunk 5]
```diff
@@ -17,7 +17,7 @@ class AppConfig {
     this.defaultTurboUrl,
     this.allowedDataItemSizeForTurbo,
     this.enableQuickSyncAuthoring = false,
-    this.enableMultipleFileDownload = false,
+    this.enableMultipleFileDownload = true,
```
Comment (karlprieb) [inline lib/services/config/app_config.dart:20]: do we want to have that enabled on prod? I don't think so

Comment (thiagocarvalhodev) [reply lib/services/config/app_config.dart:20]: No, we don't! I'll fix it

Comment (thiagocarvalhodev) [reply lib/services/config/app_config.dart]: [50a7cef](https://github.com/ardriveapp/ardrive-web/pull/983/commits/50a7cef8083fc0bb66486327291753ec178f072c)

File: lib/pages/drive_detail/components/drive_explorer_item_tile.dart (hunk 3 above)
Comment (thiagocarvalhodev) [reply lib/pages/drive_detail/components/drive_explorer_item_tile.dart]: [025a859](https://github.com/ardriveapp/ardrive-web/pull/983/commits/025a85942bcbfe289f665e27f10d008072d824fe)

File: lib/components/details_panel.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/components/details_panel.dart]: [1f3a8a0](https://github.com/ardriveapp/ardrive-web/pull/983/commits/1f3a8a0d4ef6f60c321f521f8426980c67b784a0)

**Review by karlprieb — APPROVED** [review]

**Review by karlprieb — APPROVED** [review]

**Pull request merged**